streams/src/test/java/org/apache/kafka/streams/processor/internals/CopartitionedTopicsValidatorTest.java

```diff
@@ -98,27 +98,6 @@ public void shouldSetNumPartitionsToMaximumPartitionsWhenAllTopicsAreRepartition
         assertThat(three.numPartitions, equalTo(15));
     }
 
-    @Test
-    public void shouldSetRepartitionTopicsPartitionCountToNotAvailableIfAnyNotAvaliable() {
```

Comment:
as above: should we check that an exception is thrown? (we had "infinite loop" bugs in the past -- those tests seems to be valuable)